lib/importproject.cpp

```diff
@@ -108,18 +108,34 @@ static bool simplifyPathWithVariables(std::string &s, const std::map<std::string
         if (expanded.find(var) != expanded.end())
             break;
         expanded.insert(var);
-        std::map<std::string, std::string>::const_iterator it1 = variables.find(var);
-        if (it1 == variables.end())
-            break;
-        s = s.substr(0,start) + it1->second + s.substr(end+1);
+		std::map<std::string, std::string>::const_iterator it1 = variables.find(var);
+		std::string variableValue;
+		// variable was not found within defined variables
+		if (it1 != variables.end()) {
+			variableValue = it1->second;
+		}
+		// also search environment variables
+		else {
+			char const* envValue = std::getenv(var.c_str());
+			if (envValue) {
+				variableValue = std::string(envValue);
+				// also cache value for next time to reduce system access
+				variables[var] = variableValue;
+			}
+			else {
+				break;

```

Comment:
Ideally it would be cached that "var" is not a environment variable so it won't have to run this lookup again. but then I guess one more parameter is needed and this is unfortunate also.